dev-lua/luafilesystem/luafilesystem-1.7.0.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```

Comment:
Since you most likely are going to have to fix the BDEPEND thing and rebase this anyway, you could add an extra empty line below EAPI because most of the ebuilds in tree have it so.